scenarios/validation/ArgumentsOfCorrectType.yaml

```diff
@@ -0,0 +1,26 @@
+scenario: "Validate: Argument values of correct type"
+background:
+  schema-file: validation.schema.graphql
+tests:
+  - name: correct argument type
+    given:
+      query: |
+        query BadArgumentValue {
+          dog { name(surname: true) }
+        }
+    when:
+      validate: [ArgumentsOfCorrectType]
+    then:
+      passes:
+  - name: argument type mismatch
+    given:
+      query: |
+        query BadArgumentValue {
+          dog { name(surname: "notaboolean") }
+        }
+    when:
+      validate: [ArgumentsOfCorrectType]
+    then:
+      - error-count: 1
+      - error: "Argument 'surname' of type 'Boolean' has invalid value: \"notaboolean\". Reason: Boolean value expected."
+        loc: {line: 1, column: 30}
```

Comment:
I wonder where this location is is cumming from. As far as I can tell, the error is located here:

```
  dog { name(surname: "notaboolean") }
                      ^
```

So the location would be `{line: 2, column: 23}` in this case. Have I missed something?